monitor/grafana/zeebe.json

```diff
@@ -18261,99 +18231,436 @@
                 "scaleDistribution": {
                   "type": "linear"
                 }
-              }
+              }
+            },
+            "overrides": []
+          },
+          "gridPos": {
+            "h": 6,
+            "w": 12,
+            "x": 0,
+            "y": 102
+          },
+          "id": 626,
+          "options": {
+            "calculate": false,
+            "calculation": {
+              "xBuckets": {
+                "mode": "count"
+              }
+            },
+            "cellGap": 0,
+            "color": {
+              "exponent": 0.5,
+              "fill": "dark-orange",
+              "mode": "opacity",
+              "reverse": false,
+              "scale": "exponential",
+              "scheme": "Oranges",
+              "steps": 64
+            },
+            "exemplars": {
+              "color": "rgba(255,0,255,0.7)"
+            },
+            "filterValues": {
+              "le": 1e-9
+            },
+            "legend": {
+              "show": false
+            },
+            "rowsFrame": {
+              "layout": "auto"
+            },
+            "tooltip": {
+              "mode": "single",
+              "showColorScale": false,
+              "yHistogram": false
+            },
+            "yAxis": {
+              "axisPlacement": "left",
+              "reverse": false
+            }
+          },
+          "pluginVersion": "10.4.0",
+          "targets": [
+            {
+              "datasource": {
+                "type": "prometheus",
+                "uid": "${DS_PROMETHEUS}"
+              },
+              "editorMode": "code",
+              "expr": "sum(increase(zeebe_camunda_exporter_cache_process_load_duration_failure_seconds_bucket{cluster=~\"$cluster\", namespace=~\"$namespace\", partition=~\"$partition\"}[$__rate_interval])) by (le)",
```

Comment:
:x: `increase` over `$__rate_interval` doesn't make sense. `increase` shows you the actual increase over whatever time range is used, but `$__rate_interval` is unknown so the `increase` is meaningless.
Use `rate` instead.

Reply:
I didn't touched this query, just saying.

Reply:
Ah okay, that was difficult to make out from the diff